lua/dbee/sources.lua

```diff
@@ -101,7 +101,7 @@ function sources.FileSource:create(conn)
   -- read from file
   local existing = self:load()
 
-  conn.id = "file_source_" .. self.path .. utils.random_string()
+  conn.id = "file_source_/" .. utils.random_string()
```

Comment:
This is a small quality-of-life change.

Before the change, my local notes were located in the folder i.e.
`/Users/toh995/.local/state/nvim/dbee/notes/file_source_/Users/toh995/.local/state/nvim/dbee/persistence.json0jdWVxQRAP/`

After the change, my local notes are now located in:
`/Users/toh/.local/state/nvim/dbee/notes/file_source_/0jdWVxQRAP/`

(I'm testing this on macOS btw)

Reply:
This shouldn't break existing connections, it should only affect net-new connections, which are created via the UI 😁